<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariableEditor</class>
 <widget class="QWidget" name="VariableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Variable Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="ragdollCheckBox">
          <property name="text">
           <string>Ragdoll</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="normalizedCheckBox">
          <property name="text">
           <string>Normalized</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="notVariableCheckBox">
          <property name="text">
           <string>Not Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hiddenCheckBox">
          <property name="text">
           <string>Hidden</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="outputCheckBox">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="notCharacterPropertyCheckBox">
          <property name="text">
           <string>Not Character Property</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="typeLineEdit">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Role:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="roleComboBox">
     <item>
      <property name="text">
       <string>DEFAULT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FILE_NAME</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BONE_INDEX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BONE_INDEX_MAP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EVENT_ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VARIABLE_INDEX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ATTRIBUTE_INDEX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TIME</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Default Value:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
